Firmware/ultralcd.cpp

```diff
@@ -6631,44 +6643,50 @@ static void lcd_main_menu()
     MENU_ITEM_SUBMENU_P(_i("Preheat"), lcd_preheat_menu);////MSG_PREHEAT
   }
 
-
-  if(isPrintPaused && saved_printing_type == PRINTING_TYPE_USB)
+  if (mesh_bed_leveling_flag == false && homing_flag == false && !isPrintPaused)
   {
-#ifdef FANCHECK
-      if((fan_check_error == EFCE_FIXED) || (fan_check_error == EFCE_OK))
-          MENU_ITEM_SUBMENU_P(_T(MSG_RESUME_PRINT), lcd_resume_print);////MSG_RESUME_PRINT c=18
-#else
-      MENU_ITEM_SUBMENU_P(_T(MSG_RESUME_PRINT), lcd_resume_print);////MSG_RESUME_PRINT c=18
-#endif
+	if (IS_SD_PRINTING)
```

Comment:
the code indentation looks broken - either copy the style around or use spaces on the whole new functionality

Reply:
Sorry my VScode had for some reason mix of tabs vs spaces using tabs.
Changed it to tab = 4 spaces in the function `lcd_main_menu`